src/main/java/com/solvd/micro9/vkontaktah/service/impl/PostServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.solvd.micro9.vkontaktah.service.impl;
+
+import com.solvd.micro9.vkontaktah.domain.Like;
+import com.solvd.micro9.vkontaktah.domain.Post;
+import com.solvd.micro9.vkontaktah.domain.exception.ResourceNotFoundException;
+import com.solvd.micro9.vkontaktah.persistence.PostRepository;
+import com.solvd.micro9.vkontaktah.service.PostService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class PostServiceImpl implements PostService {
```

Comment:
@eyakauleva can we delete the post?

Reply:
@h1alexbel no, I haven't implemented such logic yet